### PR TITLE
Bugfix/finfish 87 remove matplotlib

### PR DESCRIPTION
**Pull request by davemfish**
This PR removes the matplotlib dependency from invest. It was only used in the finfish model. It's functionality is not replaced with anything else -- we're just no longer generating plots as part of that model.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Unreleased Changes
 ------------------
+* The Finfish Aquaculture model know longer generates histograms for
+  uncertainity analysis due to issues with matplotlib that make invest
+  unstable. See https://github.com/natcap/invest/issues/87 for more.
```
Comment (phargogh) [inline HISTORY.rst:7]: Sorry, this is pretty trivial, but there are a couple typos: `know` --> `no` and `invest` --> `InVEST`.

Comment (davemfish) [reply HISTORY.rst:7]: Thanks, that was pretty sloppy.

**Comment by davemfish** [comment]
Thanks, @phargogh , I pushed changes to address these details. I also just looked at the User's Guide chapter and I'll now make some quick edits there to remove descriptions of the histograms that are no longer produced by finfish.

**Comment by phargogh** [comment]
Merging despite 2 flaky failing UI tests (tracking that in #36 )

**Pull request merged**